src/returns.rs

```diff
@@ -59,10 +59,15 @@ impl ReturnPass {
 
     fn emit_return_lint(&mut self, cx: &EarlyContext, spans: (Span, Span)) {
         if in_external_macro(cx, spans.1) {return;}
-        span_lint(cx, NEEDLESS_RETURN, spans.0, &format!(
-            "unneeded return statement. Consider using `{}` \
-             without the return and trailing semicolon",
-            snippet(cx, spans.1, "..")))
+        span_lint_and_then(cx, NEEDLESS_RETURN, spans.0,
```

Comment:
Previously, we said "Consider using `..`", which is nonsense. Now we say "unneeded return statement", without a suggestion.